e2e/testcases/managed_resources_test.go

```diff
@@ -75,6 +75,8 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: test-ns1
+  labels:
+    app.kubernetes.io/managed-by: configmanagement.gke.io
   annotations:
     configmanagement.gke.io/managed: enabled
```

Comment:
So if they add the annotations alone, this won't be reverted?
That seems like a behavior change. Do we care?

Reply:
Not sure if we care or not, I could write up a doc if this is controversial. The alternative is that we watch every object for each GVK, even if it's not managed by CS

Reply:
I don't think it's controversial, just want to make sure it's deliberate and acceptable and not accidental.